blob/memblob/memblob.go

```diff
@@ -332,12 +333,17 @@ func (w *writer) Close() error {
 			ContentType:        w.contentType,
 			Metadata:           w.metadata,
 			Size:               int64(len(content)),
-			ModTime:            time.Now(),
+			CreateTime:         now,
```

Comment:
Are we sure we don't just want to leave it at zero, so the user can explicitly know it is not supported?

(Also for the other drivers that don't support it)

Reply:
Done.